# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cutecom/hotfix-include.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cutecom-v0.51.0/ctrlcharacterspopup.cpp.vanilla	2025-06-29 00:22:09.237268935 +0200
+++ cutecom-v0.51.0/ctrlcharacterspopup.cpp	2025-06-29 00:22:28.312072450 +0200
@@ -23,6 +23,7 @@
 #include <QApplication>
 #include <QKeyEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPoint>
 #include <QPushButton>
 #include <QSignalMapper>
